src/utils/cert.ts

```diff
@@ -20,15 +20,14 @@ function generateCsr(keys, endpoint) {
     },
   ])
   csr.sign(keys.privateKey)
-  if (!csr.verify()) {
+  if (!csr.verify()) { // <== TODO fix
```

Comment:
help needed here

```
src/utils/cert.ts:23:12 - error TS2554: Expected 1 arguments, but got 0.

23   if (!csr.verify()) { // <== TODO fix
              ~~~~~~~~

  node_modules/@types/node-forge/index.d.ts:392:20
    392             verify(child: Certificate): boolean;
                           ~~~~~~~~~~~~~~~~~~
    An argument for 'child' was not provided.


```

Reply:
you can probably just add 
`// eslint-disable-line no-eval` since we don't want to change the functionality of that bit of code and it seems like on their github page they also use it like this